### PR TITLE
WRN-1383: Migrate Slider unit tests to Testing Library.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated tests from Enzyme to Testing Library.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed test structure to work with Testing Library.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1383

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: Slider/tests/Slider-specs.js [hunk 1]
```diff
@@ -1,512 +1,441 @@
...
-			const subject = mount(
-				<Slider min={0} max={10} onChange={handleChange} />
-			);
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should set "aria-valuetext" to hint string for the first render when vertical is true', () => {
+		render(
+			<Slider orientation="vertical" />
+		);
```
Comment (daniel-stoian-lgp) [inline Slider/tests/Slider-specs.js:32]: let's have the render statement on 1 line on every test

File: Slider/tests/Slider-specs.js [hunk 2]
```diff
@@ -1,512 +1,441 @@
...
+	});
+
+	test('should deactivate the slider on blur', () => {
+		render(
+			<Slider activateOnSelect />
+		);
+		const slider = screen.getByRole('slider');
+
+		activate(slider);
+		blur(slider);
```
Comment (daniel-stoian-lgp) [inline Slider/tests/Slider-specs.js:76]: let's have an expect slider.hasClass(active) before bluring

File: Slider/tests/Slider-specs.js [hunk 3]
```diff
@@ -1,512 +1,441 @@
...
+		activate(slider);
+		upKeyDown(slider);
+
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '51';
+
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should increment the value of vertical slider on key up when', () => {
```
Comment (daniel-stoian-lgp) [inline Slider/tests/Slider-specs.js:201]: the last word "when" seems to be extra. It's an old bug, but we can fix it now :)

Comment (alexandrumorariu) [reply Slider/tests/Slider-specs.js]: Done

File: Slider/tests/Slider-specs.js (hunk 2 above)
Comment (alexandrumorariu) [reply Slider/tests/Slider-specs.js]: Done

File: Slider/tests/Slider-specs.js (hunk 1 above)
Comment (alexandrumorariu) [reply Slider/tests/Slider-specs.js]: Done

**Review by daniel-stoian-lgp — APPROVED** [review]

**Pull request merged**